__tests__/subnet_groups.test.js

```diff
@@ -255,4 +261,20 @@ describe('subnet_groups', () => {
       expect.assertions(1);
     });
   });
+
+  describe('#buildSubnetGroups', () => {
+    it('no subnetGroups option', () => {
+      const expected = Object.keys(subnetGroupList).reduce((acc, key) => Object.assign(acc, subnetGroupList[key]), {})
+      const actual = buildSubnetGroups(2, [])
+      expect(actual).toEqual(expected);
+      expect.assertions(1);
+    });
+    it('get specific subnetGroups option', () => {
+      const getSubnetGroupsOptions = ['rds', 'redshift']
+      const expected = getSubnetGroupsOptions.reduce((acc, key) => Object.assign(acc, subnetGroupList[key]), {})
+      const actual = buildSubnetGroups(2, getSubnetGroupsOptions)
+      expect(actual).toEqual(expected);
+      expect.assertions(1);
```

Comment:
I think this code is unnecessary because no asynchronous code is here. but I just put it on your style :)